admin-dev/themes/new-theme/js/pages/order/view/order-invoices-refresher.js

```diff
@@ -55,6 +55,8 @@ export default class OrderInvoicesRefresher {
           $paymentInvoiceSelect.append(`<option value="${invoiceId}">${invoiceName}</option>`);
           $productEditInvoiceSelect.append(`<option value="${invoiceId}">${invoiceName}</option>`);
         });
+
+        document.querySelector(OrderViewPageMap.productAddInvoiceSelect).selectedIndex = 0;
```

Comment:
sadly no, its the first one... But legacy works like that so be it :)
Because the "create new invoice" option is last, so there is no other choice except changing the order of invoices froml latest to oldest i guess. Anyway, I would count it as behavior change so its not in scope.

Reply:
You're right it's not the purpose but it doesn't explode the scope either Maybe just need to change the order of returned orders, if you've got time and as long as it doesn't block the PR this could be added If decision or implementation takes too long it should be in another PR

Anyway this needs to be decided by @MatShir : should the default selected invoice be the most recent or the oldest one?

Reply:
![Screenshot from 2020-04-15 14-20-39](https://user-images.githubusercontent.com/31609858/79331799-6cda0880-7f24-11ea-8189-1bd7d2f6138b.png)